ui/apps/platform/src/Containers/Collections/CollectionFormPage.tsx

```diff
@@ -1,16 +1,90 @@
-import React from 'react';
+import React, { CSSProperties, useCallback } from 'react';
+import {
+    Breadcrumb,
+    BreadcrumbItem,
+    Button,
+    Divider,
+    PageSection,
+    Title,
+} from '@patternfly/react-core';
+
+import useRestQuery from 'Containers/Dashboard/hooks/useRestQuery';
+import { getCollection } from 'services/CollectionsService';
+import useLayoutSpaceObserver from 'hooks/useLayoutSpaceObserver';
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import { collectionsBasePath } from 'routePaths';
 import { CollectionPageAction } from './collections.utils';
+import CollectionForm from './CollectionForm';
 
 export type CollectionsFormPageProps = {
     pageAction: CollectionPageAction;
 };
 
+const noopRequest = {
```

Comment:
To keep the data fetching declarative, we use this object as a response in place of a `fetch` call when the user is on the "create" version of the form page.